manifests/backend/datastore/postgresql.pp

```diff
@@ -5,58 +5,24 @@
   include ::sensu
   include ::sensu::backend
 
-  $config_path = "${::sensu::etc_dir}/postgresql.yaml"
   $user = $::sensu::backend::postgresql_user
   $password = $::sensu::backend::postgresql_password
   $host = $::sensu::backend::postgresql_host
   $port = $::sensu::backend::postgresql_port
   $dbname = $::sensu::backend::postgresql_dbname
-  $config = {
```

Comment:
if you were specifying $config options, wouldn't those be lost?

Reply:
That was a private variable that was not accessible to anyone, it's not a parameter. It was only possible to change and the value based on parameters that have not changed. The `$config` was used to generate YAML which is no longer needed.